diff --git a/scripts/base16-material-vivid.sh b/scripts/base16-material-vivid.sh
index aa492d0..cac9765 100644
--- a/scripts/base16-material-vivid.sh
+++ b/scripts/base16-material-vivid.sh
@@ -3,7 +3,7 @@
 # Base16 Shell template by Chris Kempson (http://chriskempson.com)
 # Material Vivid scheme by joshyrobot
 
-color00="20/21/24" # Base 00 - Black
+color00="38/38/38" # Base 00 - Black
 color01="f4/43/36" # Base 08 - Red
 color02="00/e6/76" # Base 0B - Green
 color03="ff/eb/3b" # Base 0A - Yellow
@@ -11,7 +11,7 @@ color04="21/96/f3" # Base 0D - Blue
 color05="67/3a/b7" # Base 0E - Magenta
 color06="00/bc/d4" # Base 0C - Cyan
 color07="80/86/8b" # Base 05 - White
-color08="44/46/4d" # Base 03 - Bright Black
+color08="a0/c0/a0" # Base 03 - Bright Black
 color09=$color01 # Base 08 - Bright Red
 color10=$color02 # Base 0B - Bright Green
 color11=$color03 # Base 0A - Bright Yellow
@@ -21,12 +21,12 @@ color14=$color06 # Base 0C - Bright Cyan
 color15="ff/ff/ff" # Base 07 - Bright White
 color16="ff/98/00" # Base 09
 color17="8d/6e/63" # Base 0F
-color18="27/29/2c" # Base 01
-color19="32/36/39" # Base 02
+color18="28/28/28" # Base 01
+color19="21/21/21" # Base 02
 color20="67/6c/71" # Base 04
 color21="9e/9e/9e" # Base 06
 color_foreground="80/86/8b" # Base 05
-color_background="20/21/24" # Base 00
+color_background="38/38/38" # Base 00
 
 if [ -n "$TMUX" ]; then
   # Tell tmux to pass the escape sequences through
